adapters/rise/rise.go

```diff
@@ -43,7 +44,7 @@ func (a *adapter) MakeRequests(openRTBRequest *openrtb2.BidRequest, _ *adapters.
 
 	return append(requestsToBidder, &adapters.RequestData{
 		Method:  http.MethodPost,
-		Uri:     a.endpointURL + "?publisher_id=" + publisherID,
+		Uri:     a.endpointURL + "?publisher_id=" + org,
```

Comment:
Note that you are adding `org` as `publisher_id`.  Assuming this is been handled at bidder server.

Reply:
@onkarvhanumante Yes, it is 👍🏻